local-cli/server/middleware/copyToClipboard.js

```diff
@@ -0,0 +1,26 @@
+/**
+ * Copyright (c) 2015-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
+'use strict';
+
+
+module.exports = function(req, res, next) {
+  if (req.url === '/copy-to-clipboard') {
+  	var copyText = req.rawBody;
+  	if (process.platform === 'darwin') {
```

Comment:
Why not reusing https://github.com/xavi-/node-copy-paste or copying it (with proper attributions) so it works on all supported platforms?